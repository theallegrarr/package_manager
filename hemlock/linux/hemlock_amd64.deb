version https://git-lfs.github.com/spec/v1
oid sha256:da88c09f7fc4dd52e082642cc1c8e8c2a661bfe7beb32258c76c863804e3bfbc
size 81693352
