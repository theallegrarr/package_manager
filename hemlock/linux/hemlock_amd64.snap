version https://git-lfs.github.com/spec/v1
oid sha256:6f3bca75fe12733e58f417d33d7ae3df313d4819fd3afe7fb5da29f1d2f128c5
size 104525824
